FP2-blobs-out.txt

```diff
@@ -0,0 +1,427 @@
+target/product/FP2/system/vendor/lib/libmmipstreamsourcehttp.so
```

Comment:
That file is not enough. You need to add:
 - TARGET_DEVICE_BLOBS :=  FP2-blobs-out.txt
 - TARGET_DEVICE_BLOBS_LIST := true